Dockerfile

```diff
@@ -46,7 +41,7 @@ COPY elm.json elm.json
 COPY src src
```

Comment:
If there is no more executable inside the bin directory, we can get rid of the ENV line modifying the path. Can you remind me what the above code `RUN bin/build.sh && ... && ...` does again? If I remember in the other repos, we used a tar to efficiently cache elm-stuff and make elm reuse cache more often. I'm not sure anymore, and in the lightweight runner container I see both the tar archive being passed, and `elm-stuff` being copied over. Anyway these last remark do not correspond to changes in this PR, but since I had my eyes on them, I wanted to ask.

Reply:
The `RUN bin/build.sh && ...` command compiles the project, puts the compiled JS in `bin` (so we can't get rid of it, there is also `jq` and other scripts in there), runs on an exercise solution to create and save `.elm`, the analyzer `elm-stuff`  and solution `elm-stuff` necessary to run the analyzer offline.

What is the same as the test runner is that we save the reference solution `elm.json` and `elm-stuff` in a tar, and use it when running the analyzer on student solutions.
What is different is that we also save the analyzer `elm-stuff` in docker, because we also need it to run `elm-review`. We don't put it in a tar because we don't need to copy it over.